homeassistant/components/switchbee/const.py

```diff
@@ -0,0 +1,7 @@
+"""Constants for the SwitchBee Smart Home integration."""
+
+from datetime import timedelta
+
```

Comment:
It would be better to keep here only the `SCAN_INTERVAL_SEC` do the calculation to `timedelta` where you use it